PetProject/Screens/CommonViews/SearchBar.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  SearchBar.swift
+//  PetProject
+//
+//  Created by Тимофей Лукашевич on 18.04.22.
+//
+
+import SwiftUI
+
+struct SearchBar: View {
+    
+    @Binding var searchText: String
+    private var isEditing: Bool { !searchText.isEmpty }
+    var onCommit: (() -> Void)?
+    var onCancel: (() -> Void)?
+
+    var body: some View {
+        HStack {
+            TextField("Serch".localized,
```

Comment:
Try to use this style:
 ```
TextField(
   "Serch".localized,
   text: $searchText,
   onCommit: {
      self.onCommit?()
    }
)
```
might be more readable

Reply:
Done